src/index.html

```diff
@@ -1,13 +1,474 @@
+              <div class="contacts__place">
+                <a
+                  href="https://goo.gl/maps/mgzMVas13QPKXYz9A"
+                  class="contacts__address contacts__address--title"
+                  target="_blank"
+                  rel="noopener"
+                >
+                  The Met Fifth Avenue
+                </a>
+                <div class="contacts__description">
+                    Phone
+                </div>
+                <a
+                  class="contacts__address
+                         contacts__address--call"
+                    href="tel:+1 212-535-7710"
+                >
+                    +1 212-535-7710
+                </a>
+                <div class="contacts__description">
+                    Address
+                </div>
+                <a
+                  href="https://goo.gl/maps/mgzMVas13QPKXYz9A"
+                  class="contacts__address"
+                  target="_blank"
+                >
+                    1000 Fifth Avenue
+                    New York, NY 10028
+                </a>
+              </div>
+
+              <div class="contacts__place">
+                <a
+                  href="https://goo.gl/maps/mgzMVas13QPKXYz9A"
+                  class="contacts__address contacts__address--title"
+                  target="_blank"
+                >
+                  The Met Cloisters
+                </a>
+                <div class="contacts__description">
+                    Phone
+                </div>
+                <a
+                  class="contacts__address contacts__address--call"
+                    href="tel:+1 212-923-3700"
+                >
+                    +1 212-923-3700
+                </a>
+                <div class="contacts__description">
+                    Address
+                </div>
+                <a
+                  href="https://goo.gl/maps/qKQM186DAVHhXRKn7"
+                  class="contacts__address"
+                  target="_blank"
+                >
+                    99 Margaret Corbin Drive
+                    Fort Tryon Park
+                    New York, NY 10040
+                </a>
+              </div>
+            </div>
+
+            <form action="#" method="get"
+              class="contacts__form"
+              id="form">
+              <input
+                class="contacts__field"
+                type="text"
+                placeholder="Name"
+                name="name"
+                autocomplete="off"
+                required
+              >
+              <input
+                class="contacts__field"
+                type="email"
+                placeholder="Email"
+                name="email"
+                autocomplete="off"
+                required
+              >
+              <textarea
+                class="contacts__field contacts__field--textarea"
+                title="feedback"
+                name="feedback">Message
+              </textarea>
```

Comment:
```suggestion
              <textarea
                class="contacts__field contacts__field--textarea"
                title="feedback"
                name="feedback"
              ></textarea>
```
and add `Message` placeholder instead of the text "Message"
and make the field required